inline_verifier.go

```diff
@@ -0,0 +1,223 @@
+package ghostferry
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/snappy"
+	"github.com/sirupsen/logrus"
+)
+
+type InlineVerifier struct {
+	SourceDB *sql.DB
+	TargetDB *sql.DB
+
+	sourceStmtCache *StmtCache
+	targetStmtCache *StmtCache
```

Comment:
We only use the targetStmtCache for now for storing statements executing on the target. When we hook up the InlineVerifier with the BinlogStreamer, we'll need a StmtCache for the source db.

I included in here to show that I didn't refactor the StmtCache for only a single use case.